payjoin/src/core/receive/v2/mod.rs

```diff
@@ -360,8 +368,14 @@ impl Receiver<Initialized> {
         Ok((req, ohttp_ctx))
     }
 
+    /// Processes the response to the original proposal poll from the Payjoin directory.
+    ///
     /// The response can either be an UncheckedOriginalPayload or an ACCEPTED message
     /// indicating no UncheckedOriginalPayload is available yet.
+    ///
+    /// If the response contains the original proposal from the sender, transitions to the next
+    /// typestate. If the response is an ACCEPTED message from the directory which indicates that
+    /// no payload is available yet,
```

Comment:
- Imperative mood please
- Use consistent vocabulary from the BIP
- Link relevant structs

```suggestion
    /// Process the response to the Original PSBT poll from the Payjoin Directory.
    ///
    /// The response can either be an [`UncheckedOriginalPayload`] or an ACCEPTED message
    /// indicating no [`UncheckedOriginalPayload`] is available yet.
    ///
    /// If the response contains the Original PSBT from the sender, transition to the next
    /// typestate. If the response is an ACCEPTED message from the directory which indicates that
    /// no payload is available yet, continue to poll.
```